test/functional/chromedriver-e2e-specs.js

```diff
@@ -44,7 +44,7 @@ function buildReqRes(url, method, body) {
   res.status = (code) => {
     res.sentCode = code;
     return {
-      send: (body) => {
+      json: (body) => {
```

Comment:
For https://github.com/appium/appium/commit/33073c0e5fdd705a31b9c0e214b6bb49627448a7#diff-0b5330aa6f4dcf8bfd0cb010fd5f11e107a807e345694ce4390fa097644d205cL460